.licenserc.yaml

```diff
@@ -13,12 +13,13 @@ header:
     - '**/*.md'
     - '**/*.txt'
     - '.codespellignore'
-    - '.copier-answers.yml'
     - '.flake8'
     - '.jujuignore'
     - '.gitignore'
     - '.licenserc.yaml'
     - 'CODEOWNERS'
     - 'icon.svg'
     - 'LICENSE'
+    - 'files/nginx-logging-format-conf'
```

Comment:
Perhaps we should include the .conf as default too, thoughts?

Reply:
Seems like a good idea to me, several charms use .conf files here and there

Reply:
It would also be worth considering adding .json and .rule files to the defaults

Reply:
I've already added .json and .rule